R/tidyverse.R

```diff
@@ -69,14 +71,17 @@ use_tidy_ci <- function(browse = interactive()) {
   )
   use_template("codecov.yml", ignore = TRUE)
 
+  repo_type <- travis:::detect_repo_type()
+  travis:::use_tic_r(repo_type)
```

Comment:
Do we have a `tic.R` template in this package?

Reply:
Yes, for each repo type under `inst/templates`.